### PR TITLE
Don't error on missing meta when hashing untl dict.

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko, @madhulika95b, this is ready for review. This just makes it so we don't error here when we are missing a `meta` entry in the untl dict, which we do have a few occurrences of.

File: pyuntl/untldoc.py [hunk 1]
```diff
@@ -690,7 +690,7 @@ def untl_to_hash_dict(untl_elements, meaningfulMeta=True):
     show no meaningful change to metadata records.
     """
     untl_dict = untlpy2dict(untl_elements)
-    if meaningfulMeta:
+    if meaningfulMeta and untl_dict.get('meta'):
```
Comment (ldko) [inline pyuntl/untldoc.py:693]:
```suggestion
    if meaningfulMeta and untl_dict.get('meta') is not None:
```

**Review by ldko — COMMENTED** [review]
wondering if we should do None comparison?

**Comment by somexpert** [comment]
Don't see why not, I'll make that change

**Review by ldko — APPROVED** [review]
Looks good to me :+1:

**Review by madhulika95b — APPROVED** [review]
👍

**Review by ldko — APPROVED** [review]

**Pull request merged**